static/app/views/issueDetails/quickTrace/replayNode.tsx

```diff
@@ -36,6 +40,10 @@ function ReplayNode({hasReplay}: Props) {
   );
 }
 
+const StyledIcon = styled(IconPlay)<{isDarkmode: boolean}>`
+  fill: ${p => (p.isDarkmode ? p.theme.black : p.theme.white)};
```

Comment:
It'd be nice if we could use a theme alias for these colors as then we could remove the custom "isDarkMode" check, but I don't see one that exists. The closest would be `backgroundTertiary` which uses `surface100` and is #18121c for darkmode and #f5f3f7 for lightmode.

Reply:
ahh, that works well. it'd be nice to have black/white in the theme with alternates, it's hard to dig in and find these alias's